employees/views.py

```diff
@@ -1,3 +1,17 @@
-from django.shortcuts import render
+from employees.models import Report
+from employees.serializers import ReportSerializer
+from rest_framework import permissions
```

Comment:
rest_framework as side package should be at the top.

Reply:
Done